app/Http/Controllers/API/v1/SettingsController.php

```diff
@@ -162,14 +161,26 @@ public function updateSettings(Request $request): UserProfileSettingsResource|Js
         }
     }
 
+    /**
+     * Undocumented and unofficial API Endpoint
+     *
+     * @return JsonResponse
+     */
     public function deleteProfilePicture(): JsonResponse {
         if (BackendSettingsController::deleteProfilePicture(user: auth()->user())) {
-            return $this->sendResponse('', 204);
+            return $this->sendResponse(['message' => __('settings.profilePicture.deleted')]);
```

Comment:
Http code 204 missing

Reply:
changed to 200 because there is content now

Reply:
( 200 = default = missing declaration )

Reply:
![image](https://user-images.githubusercontent.com/4103693/236564161-48a33289-34fe-4b48-af04-864db5057d52.png)